pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -152,6 +152,10 @@ function validateParams(vmParams) {
         validationFailed['source'] = _("Installation Source should not be empty");
     }
 
+    if (vmParams.memorySize === 0) {
+        validationFailed['memory'] = _("Memory must be more than 0");
```

Comment:
Ups this is a reminder issue, @martinpitt asked you to change this and the conversation was marked as resolved.
s/must\ be\ more\ than/must not be/g